src/core/manager.c

```diff
@@ -1328,6 +1328,14 @@ int manager_startup(Manager *m, FILE *serialization, FDSet *fds) {
         if (r < 0)
                 return r;
 
+        if (m->first_boot && m->unit_file_scope == UNIT_FILE_SYSTEM) {
+                int e = unit_file_preset_all(UNIT_FILE_SYSTEM, 0, NULL, UNIT_FILE_PRESET_ENABLE_ONLY, NULL, 0);
+                if (e < 0)
```

Comment:
Please see CODING_STYLE: we don't declare and initialize variables in the same line, when the initialization is non-trivial, usually. Instead of declaring `e`, just reuse `r`.